helmchart/templates/ingress.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.ingress.create -}}
```

Comment:
Is this taken from the bluapi ingress? It doesn't look like the default ingress from helm create